CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Added
+
+- Hadoop: Add all necessary components to the image to mount HDFS using FUSE ([#400])
+- Java: Add needed tzdata-java package ([#425]).
```

Comment:
I like the way with putting the image name in front but can we actually go for the folder/image name e.g. java-base, stackable-base etc. ? I think this would make things clearer. (no opinion on capitalization, but imho all lower case would be fine as well)

Reply:
Done https://github.com/stackabletech/docker-images/pull/440/commits/0fc0e7c1ef6bc448a0cb10772bd516d6e780e7a3